<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Suppliers</class>
 <widget class="QDialog" name="Suppliers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Rose Garden Nursery  System 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 144, 96);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>1011</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Legal Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PACRA Reg#</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TPIN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product|Service</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone#</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Business Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bank Acc</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="The_Rose_Garden_Nursery_System">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 251, 232);
font: 700 14pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="text">
    <string> The Rose Garden Nursery  System |</string>
   </property>
  </widget>
  <widget class="QLabel" name="Add_Supplier_Detail_Section">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 251, 232);
font: 700 11pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="text">
    <string>Add Supplier Detail Section</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_View_Supplier">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 221, 218);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>View Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Insert_Supplier">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(19, 255, 0);
font: 500 15pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Update_Supplier">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 95, 180);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Delete_Supplier">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SupplierID">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_Legal_Name">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>120</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_Pacra_Reg">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_Tpin">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>220</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_Product_Service">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>280</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_Phone">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>330</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_Business_Address">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_ID">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>40</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ID </string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_ID">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>40</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_Legal_Name">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Legal Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_Pacra_Registration_Number">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>150</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PACRA Registration number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_TPIN">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>200</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TPIN | Tax Identification Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_NAME">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>100</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_PACRA">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>150</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_TPIN">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>200</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_Product_Service">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>260</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Product | Service</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_Product_Service">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_Phone">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>310</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_Phone">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_Business_Address">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Business Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_Address">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>360</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="Supplier_Bank_Acc">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>410</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Bank Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="Asteric_B_Acc">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>410</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Supplier_B_Acc">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>430</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
